packages/execution-environments/src/common/commands.ts

```diff
@@ -71,7 +71,7 @@ export function getCommandMethodImplementations(
         throw new Error('request is not a proper JSON RPC Request');
       }
 
-      return invokeSnapRpc(target, origin, request);
+      return (await invokeSnapRpc(target, origin, request)) ?? null;
```

Comment:
What's the reasoning for adding the `await` here?

Reply:
`invokeSnapRpc` returns values from the Snap's `onRpcRequest`. These values can be `undefined`, which is not a valid JSON value, and so we now just default to `null` in those cases.

Reply:
Makes sense 👍